tasks/main.yaml

```diff
@@ -13,10 +15,26 @@
     baseurl: "{{ rabbitmq_epel_yumrepo_url }}"
     gpgkey: "{{ rabbitmq_epel_yumrepo_gpgkey }}"
     params: "{{ rabbitmq_epel_yumrepo_params }}"
-  when: rabbitmq_epel_install == true
+  when: >
+    rabbitmq_epel_install == true and
+    ansible_os_family == "RedHat" and
+    ansible_distribution_major_version | int >= 6
   tags:
     - rabbitmq_pkg
 
+- name: Add apt key
+  apt_key:
+    url: "{{ rabbitmq_debian_aptrepo_key }}"
+    state: "present"
+  when: ansible_os_family == "Debian"
+
+- name: Add apt repository
+  apt_repository:
+    repo: "{{ debian_apt_repository }}" 
+    state: present
+    update_cache: true
```

Comment:
Please remove this line.

Reply:
I tested after removing the update_cache line and the installation fails without it

Reply:
That's weird as I'm using the `package` module without it in other roles (e.g. https://github.com/jtyr/ansible-kibana).

Reply:
Apologies, I must have tested it wrong the first time.  I just tested it again and it's working without the update_cache requirement.